rosdep/python.yaml

```diff
@@ -5899,6 +5899,13 @@ python3-empy:
   openembedded: [python3-empy@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-empy']
   ubuntu: [python3-empy]
+python3-evdev:
+  debian: [python3-evdev]
+  fedora: [python3-evdev]
+  osx:
+    pip:
```

Comment:
I think we prefer to avoid mixing pip packages with system packages under the same rosdep key. So I think this should be moved to it's own rule named `python3-evdev-pip`. It would be good to get someone else to confirm.

Reply:
It's fine to have pip rules to fill in alternative platforms and older distros where there's not expected to be a conflicting native package in the future.